plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/RepositoryList.js

```diff
@@ -32,12 +32,13 @@ Ext.define('NX.coreui.view.repository.RepositoryList', {
       width: 36,
       iconVariant: 'x16',
       iconNamePrefix: 'repository-',
-      dataIndex: 'type'
+      dataIndex: 'type',
+      id: 'icon'
     },
-    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_NAME_COLUMN'), dataIndex: 'name', flex: 1 },
-    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_TYPE_COLUMN'), dataIndex: 'type' },
-    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_FORMAT_COLUMN'), dataIndex: 'format' },
-    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_STATUS_COLUMN'), dataIndex: 'status', flex: 1, 
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_NAME_COLUMN'), dataIndex: 'name', id: 'name', flex: 1 },
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_TYPE_COLUMN'), dataIndex: 'type' , id: 'type' },
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_FORMAT_COLUMN'), dataIndex: 'format', id: 'format' },
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_STATUS_COLUMN'), dataIndex: 'status', id: 'status', flex: 1, 
```

Comment:
we may have to look at this closer then, as id is globally unique to the entire app.  So you are saying the id='name' component for the entire app is a column.

Looks like columns have a stateId that should be used for what you are attempting:

http://docs-origin.sencha.com/extjs/4.2.3/#!/api/Ext.grid.column.Column-cfg-stateId